src/plugins/controls/public/time_slider/components/time_slider_prepend.tsx

```diff
@@ -72,28 +93,16 @@ export const TimeSliderPrepend: FC<Props> = (props: Props) => {
         }}
         iconType="framePrevious"
         color="text"
-        aria-label={i18n.translate('controls.timeSlider.previousLabel', {
-          defaultMessage: 'Previous time window',
-        })}
+        aria-label={TimeSliderStrings.control.getPreviousButtonAriaLabel()}
         data-test-subj="timeSlider-previousTimeWindow"
       />
-      {props.waitForControlOutputConsumersToLoad$ === undefined ? null : (
-        <EuiButtonIcon
-          className="timeSlider-playToggle"
-          onClick={isPaused ? onPlay : onPause}
-          iconType={isPaused ? 'playFilled' : 'pause'}
-          size="s"
-          display="fill"
-          aria-label={
-            isPaused
-              ? i18n.translate('controls.timeSlider.playLabel', {
-                  defaultMessage: 'Play',
-                })
-              : i18n.translate('controls.timeSlider.pauseLabel', {
-                  defaultMessage: 'Pause',
-                })
-          }
-        />
+      {props.waitForControlOutputConsumersToLoad$ ===
+      undefined ? null : showApplySelectionsButton ? (
+        <EuiToolTip content={TimeSliderStrings.control.getPlayButtonDisabledTooltip()}>
+          {PlayButton}
+        </EuiToolTip>
+      ) : (
+        PlayButton
```

Comment:
Currently, the time slider play button is **disabled** when `showApplySelections` is `true` - and to make it clear why, I added the following tooltip:

![image](https://github.com/elastic/kibana/assets/8698078/b5cb7fbf-8b66-412a-8291-9639701ff18c)


I wonder though - this tooltip isn't particularly helpful for dashboard **viewers**, and only really provides context to the authors.... Should we simply hide the play button when `showApplySelections` is `true` +  the dashboard is in view mode? 🤔

Reply:
@andreadelrio

Reply:
I agree with hiding it in view mode.

Reply:
Done in 5aca279ab1f92a7b0e34fea0150202fcd99abe56 :+1: